_episodes_rmd/02-func-R.Rmd

```diff
@@ -39,7 +39,7 @@ Let's start by defining a function `fahrenheit_to_kelvin` that converts temperat
 
 ```{r}
 fahrenheit_to_kelvin <- function(temp_F) {
-  temp_K <- ((temp_F - 32) * (5 / 9)) + 273.15
+  temp_K <- (temp_F + 459.67) * 5 / 9
```

Comment:
Because [my co-maintainer seems to agree](https://github.com/swcarpentry/r-novice-inflammation/pull/440#issuecomment-517583426), please change your F-based variant into a code comment and re-insert the C-based line above.

That probably achieves our common goal of _not confusing_ learners with at-the-moment irrelevant information as a [minimum viable change](https://about.gitlab.com/handbook/values/#minimum-viable-change-mvc).